app/models/resource.rb

```diff
@@ -14,4 +15,26 @@ class Resource < ApplicationRecord
   has_many :resource_dbs
   has_many :dbs, through: :resource_dbs
   belongs_to :project
+
+
+  private 
+
+  def images_validation
+    error_message = ''
+    images_valid = true
+    if images.attached?
+      images.each do |image|
+        if !image.blob.content_type.in?(%w(image/jpeg image/jpg image/png))
+          images_valid = false
+          error_message = "can't have this format"
```

Comment:
This is a general thing but try to tell users what they _can_ do rather than what they can't, e.g. "must be in x, y, z format"